### PR TITLE
Update paphos info

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: docs/machines/paphos.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* Drives: Internal SATA DVD+/-RW
+* Network: NetXtreme II BCM5709 Gigabit Ethernet
+
+## Where to find
+
+* 136.206.15.26
+
+## Services
+
+* Mail Server
```
Comment (butlerx) [inline docs/machines/paphos.md]: - ldap
 - dns
 - list
 - irc
 - bitlbee

**Review by butlerx — CHANGES_REQUESTED** [review]
@zergless what services am i forgetting

File: docs/machines/paphos.md (hunk 1 above)
Comment (zergless) [reply docs/machines/paphos.md]: mailing lists instead of just list pls

Comment (butlerx) [reply docs/machines/paphos.md]: could you change mail server so its 
- Mail
  - exim
  - dovecoat

Comment (butlerx) [reply docs/machines/paphos.md]: also icecast and boards runs on paphos could you also link these to their entries

**Review by butlerx — CHANGES_REQUESTED** [review]

**Comment by zergless** [comment]
I think you got most of them

File: docs/machines/paphos.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Where to find
+
+* 136.206.15.26
+
+## Services
+
+* Mail 
+    - [exim](/services/exim)
+    - dovecoat
```
Comment (butlerx) [inline docs/machines/paphos.md]: why not link dovecoat? /services/dovecot

Comment (gruunday) [reply docs/machines/paphos.md]: Can't find a doc entry for it. But then again am tired

Comment (butlerx) [reply docs/machines/paphos.md]: services/dovecot

Comment (gruunday) [reply docs/machines/paphos.md]: huh well

File: docs/machines/paphos.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+* 136.206.15.26
+
+## Services
+
+* Mail 
+    - [exim](/services/exim)
+    - [dovecoat](services/dovecot)
+* [ldap](/services/ldap)
+* [dns](/services/bind9)
```
Comment (butlerx) [inline docs/machines/paphos.md:22]: is bind not in netwrok

Comment (VoyTechnology) [reply docs/machines/paphos.md:22]: It is unless #12 is merged

Comment (gruunday) [reply docs/machines/paphos.md:22]: Well until that one is resolved i'll leave this here

Comment (butlerx) [reply docs/machines/paphos.md:22]: #12 is not merged and still in discussion

**Review by butlerx — APPROVED** [review]

**Pull request merged**